lib/dynflow/telemetry_adapters/abstract.rb

```diff
@@ -0,0 +1,71 @@
+module Dynflow
+  module TelemetryAdapters
+    class Abstract
+      # Default buckets to use when defining a histogram
+      DEFAULT_BUCKETS = [10, 50, 200, 1000, 15_000].freeze
```

Comment:
These need to be in seconds.

Reply:
This were the buckets so far I was testing with, that seemed like good distrubution for the tasks timing:

```
[0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 20, 30, 60, 120, 300, 600, 1200]
```

Reply:
Yeah all credit goes to @iNecas who found it. I had no idea.

FYI statsd does not have any notion of buckets, this is for prometheus exporter, you don't need them in your codebase, in core it is part of stack because then there is a YAML exporter generator, here you have your YAML hardcoded - so drop this line, it's useless.

Reply:
The yaml from the repository is used only when running "standalone Dynflow". When Dynflow is used with Foreman, these values should still get picked up by the yaml exporter generator. That's how the hardcoded yaml in this repo was created in the first place